### ISSUE TITLE
PowerGrids OneBusOneLoad fails with NB because of not evaluated parameter

### ISSUE THREAD

**Issue by casella** [OpenModelica/OpenModelica#10014]
Please run [PowerGrids.Electrical.Test.OneBusOneLoad](https://libraries.openmodelica.org/branches/newInst-newBackend/PowerGrids/files/PowerGrids_PowerGrids.Electrical.Test.OneBusOneLoad.sim) with `--newBackend`. The model is compiled successfully, but the following runtime error is obtained:
```
LOG_SUCCESS       | info    | The initialization finished successfully without homotopy method.
assert            | debug   | division by zero at time 0.01, (a=0) / (b=0), where divisor b expression is: infiniteBus.port.IBase
assert            | debug   | division by zero at time 0, (a=0) / (b=0), where divisor b expression is: infiniteBus.port.IBase
```
`IBase` should be computed to 577.35.

**Comment by casella** [comment]
At the moment #11556 prevents getting to the simulation phase. I'm keeping this open so we can check again when that is solved.

**Comment by casella** [comment]
#11556 was fixed by #11827. The test OneBusOneLoad now [runs fine](https://libraries.openmodelica.org/branches/newInst-newBackend/PowerGrids/files/PowerGrids_PowerGrids.Electrical.Test.OneBusOneLoad.sim).
